gensim/summarization/bm25.py

```diff
@@ -116,6 +119,13 @@ def _initialize(self, corpus):
                 negative_idfs.append(word)
         self.average_idf = float(idf_sum) / len(self.idf)
 
+        if self.average_idf < 0:
+            logger.warning(
+                'Average inverse document frequency is less than zero. Your corpus of {} documents'
+                ' is either too small or it does not originate from actual text documents. BM25'
+                ' will likely produce "wrong" results.'.format(self.corpus_size)
```

Comment:
This `"wrong"` sounds almost mystical :) Any way we can make this more concrete, avoid the scare quotes?

Reply:
Rigor does not always improve clarity. See my other comment and let me know if you have an idea how to better rephrase this.

Reply:
```suggestion
                ' will likely produce unintuitive results.'.format(self.corpus_size)
```